src/Raven.Server/Config/Categories/StudioConfiguration.cs

```diff
@@ -1,13 +1,78 @@
+using System;
 using System.ComponentModel;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
 using Raven.Server.Config.Attributes;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.ServerWide;
 
 namespace Raven.Server.Config.Categories
 {
     public class StudioConfiguration : ConfigurationCategory
     {
-        [Description("The directory into which RavenDB will search the studio files, defaults to the base directory")]
+        [Description("The directory in which RavenDB will search the studio files, defaults to the base directory")]
         [DefaultValue(null)]
         [ConfigurationEntry("Studio.Path", ConfigurationEntryScope.ServerWideOnly)]
         public string Path { get; set; }
+        
+        [Description("Semicolon seperated list of notification names which will not be shown in the Studio. If not specified, all notifications are allowed. Example: \"SlowIO;Server_NewVersionAvailable;ClusterTopologyWarning\".")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Studio.Notifications.FilterOut", ConfigurationEntryScope.ServerWideOnly)]
+        public string[] FilteredOutNotifications { get; set; }
```

Comment:
You are always comparing on this, easier if you will use `HashSet<string>` (case insensitive)